cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,54 @@
+/* eslint-disable max-len */
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
-  before(() => {
-
+  let user;
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it('', () => {
+  it('should register an account with all fields filled except the "picture" field', () => {
+    cy.findByPlaceholder('First Name').type(user.firstName);
+    cy.findByPlaceholder('Last Name').type(user.lastName);
+    cy.findByPlaceholder('name@example.com').type(user.email);
+    cy.get(`#gender-radio-2`).click({ force: true });
```

Comment:
why did you write randomGenderIndex if you didn't use it? And your assertion becomes hardcoded because of it

Reply:
I mixed up how I could assert gender and hobbies, but now it fixed